cmd/up/up.go

```diff
@@ -249,16 +253,18 @@ func Up() *cobra.Command {
 			}
 
 			up := &upContext{
```

Comment:
I can't test this path given that there is a big refactor needed in order to modify RunE